lib/streamlit/elements/lib/utils.py

```diff
@@ -175,9 +181,42 @@ def compute_and_register_element_id(
     The element ID gets registered to make sure that only one ID and user-specified
     key exists at the same time. If there are duplicated IDs or keys, an error
     is raised.
+
+    Parameters
+    ----------
+    element_type : str
+        The type (command name) of the element to register.
+
+    user_key : str | None
+        The user-specified key for the element. `None` if no key is provided
+        or if the element doesn't support a specifying a key.
+
+    form_id : str | None
+        The ID of the form that the element belongs to. `None` or empty string
+        if the element doesn't belong to a form or doesn't support forms.
+
+    kwargs : SAFE_VALUES | Iterable[SAFE_VALUES]
+        The arguments to use to compute the element ID.
+        The arguments must be stable, deterministic values.
```

Comment:
I would add a comment here that some context variables, e.g. the script_hash, will be added by this function and do not have to be passed by the caller.

Reply:
Yep, I can add this. I think the test I added automatically makes sure that the previous `page` variable isn't used anymore.

Reply:
Added 👍